manifests/init.pp

```diff
@@ -215,17 +220,20 @@
   }
 
   if $pulpcore_mirror {
-    $pulpcore_allowed_import_path    = ['/var/lib/pulp/sync_imports']
-    $pulpcore_allowed_export_path    = []
+    $base_allowed_import_path    = ['/var/lib/pulp/sync_imports']
+    $base_allowed_export_path    = []
```

Comment:
Might as well makes these plural as well since they are arrays and you are introducing a new name.